dotCMS/src/main/java/com/dotmarketing/util/ImportExportUtil.java

```diff
@@ -1428,6 +1428,7 @@ private void doXMLFileImport(File f, PrintWriter out, ObjectFilter filter)throws
                         }
                         else if(obj instanceof MultiTree){
                             final MultiTree t = (MultiTree) obj;
+                            t.setPersonalization(MultiTree.DOT_PERSONALIZATION_DEFAULT);
```

Comment:
This will prevent us from importing starters that have personalization already.  I think you should incorporate the logic into a lower level, e.g. the MultiTreeFactory db save.  Something like:  
```java
if(!UtilMethods.isSet(tree.getPersonalization()){
   tree.setPersonalization(MultiTree.DOT_PERSONALIZATION_DEFAULT);
}

Reply:
This has been done in the another branch